.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 sudo: false
 node_js:
-  - '0.10'
+  - '6.11.3'
```

Comment:
we can have multiple versions of node in here, and would suggest we have:
- the minimum node version we support
- any later stable node versions (ie 8) also

Including the latter will help us diagnose any potential issues between the minimum version we support and later versions of node (ie breaking changes, deprecated methods between versions)